packages/testcontainers/src/index.ts

```diff
@@ -14,15 +14,15 @@ export const GenesisKeys = RegTestFoundationKeys
  */
 export type MasterNodeKey = MNK
 
-export * from './containers/DeFiDContainer'
-export * from './containers/MainNetContainer'
-export * from './containers/TestNetContainer'
-export * from './containers/RegTestContainer/index'
-export * from './containers/RegTestContainer/Masternode'
-export * from './containers/RegTestContainer/Persistent'
-export * from './containers/RegTestContainer/ContainerGroup'
+export * from './containers/Dockerode/DeFiDContainer'
+export * from './containers/Dockerode/MainNetContainer'
+export * from './containers/Dockerode/TestNetContainer'
+export * from './containers/Dockerode/RegTestContainer/index'
+export * from './containers/Dockerode/RegTestContainer/Masternode'
+export * from './containers/Dockerode/RegTestContainer/Persistent'
+export * from './containers/Dockerode/RegTestContainer/ContainerGroup'
 export * from './utils'
 
-export * from './containers/RegTestContainer/LoanContainer'
+export * from './containers/Dockerode/RegTestContainer/LoanContainer'
 
-export * from './containers/AppContainer/WhaleSanityContainer'
+export * from './containers/Dockerode/AppContainer/WhaleSanityContainer'
```

Comment:
To reduce the scope of change and review fatigue, should not change files that don't need to be changed, especially for this PR, since it should act more as an addition over modification or replacement. It would also help to keep the commit history intact.